bricks/test_optimizer/CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+## 🦄 Contributing to Very Good CLI
+
+First of all, thank you for taking the time to contribute! 🎉👍 Before you do, please carefully read this guide.
+
+## Developing for Very Good CLI's test optimizer
+
+To develop for Very Good CLI's test optimizer, you will also need to become familiar with our processes and conventions detailed [here](../../CONTRIBUTING.md).
+
+💡 **Note**: The test optimizer brick is not published at [Brick Hub](brickhub.dev). It is not intended to be used by the general public. Instead, it has been designed to work closely with Very Good CLI's `test` command.
+
+### Setting up your local development environment
+
+1. Install a valid [Dart SDK](https://dart.dev/get-dart) in your local environment. Compatible Dart SDK versions with test optimizer can be found [here](https://github.com/VeryGoodOpenSource/very_good_cli/blob/main/bricks/test_optimizer/hooks/pubspec.yaml). If you have Flutter installed you likely have a valid Dart SDK version already installed.
+
+2. Install [Mason](https://github.com/felangel/mason/tree/master/packages/mason_cli#installation) in your local environment:
+
+```sh
+# 🎯 Activate from https://pub.dev
+dart pub global activate mason_cli
+```
+
+3. Get hooks' dependencies:
+
+```sh
+# 🪝 Get hooks' dependencies (from bricks/test_optimizer/hooks)
+dart pub get
+```
+
```

Comment:
```suggestion
3. Get hooks dependencies:

```sh
# 🪝 Get hooks dependencies (from bricks/test_optimizer/hooks)
dart pub get
```

```
maybe i'm misreading, but i don't think this needs to be possessive